ndcube/asdf/resources/schemas/ExtraCoords-0.1.0.yaml

```
@@ -0,0 +1,37 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/ExtraCoords-0.1.0"
```

I suggest we stick with lowercase with `_` separators for schema uris. This helps to avoid issues with filenames vs uris with case-insensitive filesystems.
```suggestion
id: "asdf://sunpy.org/ndcube/schemas/extra_coords-0.1.0"
```